include/onnxruntime/core/providers/dml/dml_provider_factory.h

```diff
@@ -30,6 +30,23 @@
 extern "C" {
 #endif
 
+enum class OrtDmlPerformancePreference {
+  Default,
```

Comment:
be explicit about the values here

Default = 0
HighPerformance = 1
LowPower = 2